src/views/docs/en/guides/developer-experience/local-development.md

```diff
@@ -4,112 +4,103 @@ category: Developer experience
 
-```javascript
-const test = require('tape')
-const tiny = require('tiny-json-http')
-const sandbox = require('@architect/sandbox')
+The following structure was added your project:
+<!-- unsure if this diagram is necessary -->
+```
+.
+├── src
+│   └── events
+│       └── hit-counter
+│           ├── index.js
+│           └── config.arc
+│
+└── app.arc
+```
 
-test('setup', async t=> {
-  t.plan(1)
-  await sandbox.start()
-  t.ok(true, 'sandbox started on http://localhost:3333')
-})
+### Add dependencies (optional)
 
-test('get /', async t=> {
-  t.plan(1)
-  let result = await tiny.get({ url: 'http://localhost:3333' })
-  t.ok(result, 'got 200 response')
-})
+Your new event may require some dependencies. For this example, [@architect/functions](https://github.com/architect/functions) will be helpful to handle the event subscription. Dependencies can be installed directly to the `hit-counter` directory:
 
-test('teardown', async t=> {
-  t.plan(1)
-  await sandbox.end()
-  t.ok(true, 'sandbox ended')
-})
+1. Inside `./src/events/hit-counter` create a `package.json` file with an empty `dependencies` entry:
+
+```json
+// package.json
+{
+  "dependencies": {}
+}
 ```
 
-### Testing `@events` and `@queues`
-
-Use `@architect/functions` to publish locally to `@events` or `@queues` for testing.
-
-```javascript
-const test = require('tape')
-const arc = require('@architect/functions')
-const sandbox = require('@architect/sandbox')
-
-test('setup', async t=>{
-  t.plan(1)
-  await sandbox.start()
-  t.ok(true, 'started')
-})
-
-test('@events', async t=> {
-  t.plan(1)
-  // mock pingID
-  let pingID = 'testing-ping'
-  // send a ping event
-  await arc.events.publish({
-    name: 'ping',
-    payload: { pingID }
-  })
-  // see if testing-ping is in the db
-  let data = await arc.tables()
-  let { hits } = await data.pings.get({ pingID })
-  t.ok(hits, 'pong!')
-})
-
-test('teardown', async t=>{
-  t.plan(1)
-  await sandbox.end()
-  t.ok(true, 'closed')
-})
+2. Install dependencies with `npm`:
 
+```console
+cd src/events/hit-counter
+npm install @architect/functions
 ```
 
-### Testing `@tables`
-
-Using `@architect/functions` to interact with DynamoDB `@tables` defined locally.
-
-```javascript
-const test = require('tape')
-const arc = require('@architect/functions')
-const sandbox = require('@architect/sandbox')
-
-test('setup', async t=>{
-  t.plan(1)
-  await sandbox.start()
-  t.ok(true, 'started')
-})
-
-test('db', t=> {
-  t.plan(1)
-  let data = await arc.tables()
-  let cats = await data.cats.scan({})
-  t.ok(Array.isArray(cats.Items), 'bag o cats')
-})
-
-test('teardown', async t=>{
-  t.plan(1)
-  await sandbox.end()
-  t.ok(true, 'closed')
-})
+3. From the project root start up the sandbox if it is not already running:
+
+```console
+arc sandbox
 ```
+
+Architect will hydrate your [shared code](/docs/en/guides/developer-experience/sharing-code) and run the updated Architect project.
```

Comment:
We didn't mention adding shared code as part of this local dev guide, so this is a bit awkward. But it'd def make sense to demonstrate shared code.

Reply:
Agreed, perhaps we can reword this to state that _sandbox_ will hydrate function code and link to the `arc hydrate` docs while we are at it?

```suggestion
[`arc sandbox`](../../reference/cli/sandbox) will automatically [hydrate - that is, install needed dependencies](../../reference/cli/hydrate) - for every function making up your Architect project.
```